src/redux/actions/samples/createSample.js

```diff
@@ -60,9 +62,13 @@ const createSample = (
     if (type === '10X Chromium') {
       sampleTechnology = '10x';
     } else {
-      throw new Error(`Sample type ${type} is not implemented`);
+      throw new Error(`Sample technology ${type} is not recognized`);
     }
```

Comment:
I added it for v2 because are doing this if that sets the string to `10x` so it is more likely to impact api v2, it isn't really a problem in api v1 becuase we don't really ever check the type after the initial load